### PR TITLE
Upgrade ruff to 0.0.262

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by eladkal — APPROVED** [review]

File: airflow/providers/google/cloud/hooks/bigtable.py [hunk 1]
```diff
@@ -248,7 +248,9 @@ def delete_table(self, instance_id: str, table_id: str, project_id: str) -> None
         """
         instance = self.get_instance(instance_id=instance_id, project_id=project_id)
         if instance is None:
-            raise RuntimeError("Instance %s did not exist; unable to delete table %s" % instance_id, table_id)
+            raise RuntimeError(
+                "Instance {} did not exist; unable to delete table {}".format(*instance_id), table_id
```
Comment (josh-fell) [inline airflow/providers/google/cloud/hooks/bigtable.py]: Should we just use f-strings here instead?

**Review by josh-fell — APPROVED** [review]

File: airflow/providers/google/cloud/hooks/bigtable.py (hunk 1 above)
Comment (potiuk) [reply airflow/providers/google/cloud/hooks/bigtable.py]: Why not

Comment (potiuk) [reply airflow/providers/google/cloud/hooks/bigtable.py]: (this has been automatically converted by ruff, so this is interesting change.

Comment (potiuk) [reply airflow/providers/google/cloud/hooks/bigtable.py]: It lookes like Ruff treated table_id as linked exception :)

Comment (potiuk) [reply airflow/providers/google/cloud/hooks/bigtable.py]: AH.... That WAS a bug :)

It should have been:

```
raise RuntimeError("Instance %s did not exist; unable to delete table %s" % (instance_id, table_id))
```

**Comment by potiuk** [comment]
Need https://github.com/apache/airflow/pull/30810 to be merged (crossing PRs broke main).

**Pull request merged**